### ISSUE TITLE
single-variable prediction

### ISSUE THREAD

**Issue by Pyramiding** [oliverguhr/transformer-time-series-prediction#27]
This seems to be a single-variable prediction, which only uses the sequence information of the own variables of the time signal,  does not use other features

**Comment by oliverguhr** [comment]
Correct, this is a univariate model. But you can adjust this to multi variate scenarios.

**Comment by Pyramiding** [comment]
got it
